android/app/src/main/java/org/fossasia/openevent/api/Urls.java

```diff
@@ -46,6 +50,12 @@ public static String getBaseUrl() {
     }
 
     public static void setBaseUrl(String baseUrl) {
-        BASE_URL = baseUrl;
+        if (URLUtil.isValidUrl(baseUrl)) {
```

Comment:
Yes @the-dagger, you are correct. We can never know if an api is correct without actually making a request. This PR focuses on invalid api urls which was making the app crash